### PR TITLE
fix: otel metric semantic convs

### PR DESCRIPTION
**Pull request by toddbaert**
The keys were wrong :grimacing:

### CODE REVIEW

**Comment by beeme1mr** [comment]
Perhaps you should move these to the const file since it's the same between metrics and spans.

https://github.com/open-feature/js-sdk-contrib/blob/main/libs/hooks/open-telemetry/src/lib/constants.ts

**Comment by toddbaert** [comment]
> Perhaps you should move these to the const file since it's the same between metrics and spans.
> 
> https://github.com/open-feature/js-sdk-contrib/blob/main/libs/hooks/open-telemetry/src/lib/constants.ts

yep

**Comment by toddbaert** [comment]
@beeme1mr updated.

**Review by Kavindu-Dodan — APPROVED** [review]

File: libs/hooks/open-telemetry/src/lib/attributes.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export const ACTIVE_COUNT_NAME = `${FEATURE_FLAG}.evaluation_active_count`;
+export const REQUESTS_TOTAL_NAME = `${FEATURE_FLAG}.evaluation_requests_total`;
+export const SUCCESS_TOTAL_NAME = `${FEATURE_FLAG}.evaluation_success_total`;
+export const ERROR_TOTAL_NAME = `${FEATURE_FLAG}.evaluation_error_total`;
+
+export type EvaluationAttributes = {[key: `${typeof FEATURE_FLAG}.${string}`]: string | undefined };
+export type ExceptionAttribute = { [key in `${typeof FEATURE_FLAG}.exception`]: string };
+
+export const KEY_ATTR: keyof EvaluationAttributes = `${FEATURE_FLAG}.key`;
```
Comment (Kavindu-Dodan) [inline libs/hooks/open-telemetry/src/lib/attributes.ts:11]: nit - can refer to FF sem conv documentation for first three attributes : https://opentelemetry.io/docs/specs/otel/logs/semantic_conventions/feature-flags/

Comment (toddbaert) [reply libs/hooks/open-telemetry/src/lib/attributes.ts:12]: Added this link in a comment.

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**